File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_set_discharge_time.rs


pub fn convert__command__set_discharge_time__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command::SetDischargeTime) -> crate::victron_controller::dashboard::command::SetDischargeTime {
    crate::victron_controller::dashboard::command::SetDischargeTime {
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_command_set-discharge-time.ts
// @ts-nocheck
import {SetDischargeTime as v0_1_0_SetDischargeTime} from '../v0_1_0/Command'
import {SetDischargeTime as v0_2_0_SetDischargeTime} from './Command'

export function convert__command__set_discharge_time__from__0_1_0(from: v0_1_0_SetDischargeTime): v0_2_0_SetDischargeTime {
    return new v0_2_0_SetDischargeTime (
        JSON.parse(JSON.stringify(from.value))
    )
}